monkey/agent_plugins/exploiters/snmp/src/plugin.py

```diff
@@ -42,7 +44,7 @@ def __init__(
         otp_provider: IAgentOTPProvider,
         **kwargs,
     ):
-        exploit_client = StubSNMPExploitClient(agent_id, agent_event_publisher)
+        exploit_client = SNMPExploitClient(agent_id, agent_event_publisher, plugin_name)
```

Comment:
I don't see where SNMPExploitClient is using `plugin_name`

Reply:
It's using it for the propagation/exploitation events (it's called `exploiter_name` in the class)